src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs

```diff
@@ -334,22 +283,13 @@ public Block Process(Block suggestedBlock, ProcessingOptions options, IBlockTrac
 
             ProcessingBranch processingBranch = PrepareProcessingBranch(suggestedBlock, options);
             PrepareBlocksToProcess(suggestedBlock, options, processingBranch);
-
-            try
+            Block[] processedBlocks = ProcessBranch(
+                options | ProcessingOptions.RerunWithTraceOnFailure,
```

Comment:
Do we want to force it? Why put it in ProcessingOptions in first place then?

Reply:
because it calls itself recursively we had to just force it on the top level